Header header
nav_msgs/Odometry odometry   # robot odometry in global frame
nav_msgs/Odometry last_odometry   # robot odometry in global frame when it detects the marker last time
MGSMeasurement[] measurements   # detection measurements
int8 GAP_FLAG
